### ISSUE TITLE
How do I play at 960x540 in-game resolution but with a 1920x1080 desktop resolution?

### ISSUE THREAD

**Issue by HugeTaffer** [alexbatalov/fallout1-ce#101]
I'd like to play as if I set 960x540 in the f1-res.ini and 1920x1080 in ddraw.ini, is it possible in this build to do this?

**Comment by a-hurst** [comment]
@HugeTaffer Yes, just set `SCALE_2X=1` in f1_res.ini along with the resolution set to 1920x1080. Note that you'll need to use the latest development release for this to work, as `SCALE_2X` support was added since the last official release.

**Comment by HugeTaffer** [comment]
Thanks!

**Comment by forb1d** [comment]
@a-hurst could you please guide me on where / how to get "the latest development release". Thank you and sorry for the naive question!
